src/registrar/management/commands/utility/terminal_helper.py

```diff
@@ -1,8 +1,25 @@
+from enum import Enum
 import logging
 import sys
 
 logger = logging.getLogger(__name__)
 
+class LogCode(Enum):
+    """Stores the desired log severity
+    
+    Overview of error codes:
+    - 1 ERROR
+    - 2 WARNING
+    - 3 INFO
+    - 4 DEBUG
+    - 5 DEFAULT
+    """
+
+    ERROR = 1
+    WARNING = 2
+    INFO = 3
+    DEBUG = 4
+    DEFAULT = 5
```

Comment:
We throw a lot of logs around when running these scripts, especially when parsing additional data. This enumerator allows us to keep tabs on what "kind" of error was thrown. 

Default specifies that no log severity was defined. At that point, it is up to the function to decide how to handle it.